src/ResourceManager/RecoveryServices.SiteRecovery/Commands.RecoveryServices.SiteRecovery/help/Update-AzureRmRecoveryServicesAsrProtectionDirection.md

```diff
@@ -73,6 +92,18 @@ The **Update-AzureRmRecoveryServicesAsrProtectionDirection** cmdlet updates the
 PS C:\> $currentJob = Update-AzureRmRecoveryServicesAsrProtectionDirection -RecoveryPlan $RP -Direction PrimaryToRecovery
 ```
 
+### Example 2
+```
+PS C:\> $currentJob = Update-AzureRmRecoveryServicesAsrProtectionDirection -AzureToAzure -ProtectionContainerMapping $B2ApcmMapping -LogStorageAccountId $cachceStorageId
+ -ReplicationProtectedItem $rpi
+```
+
+### Example 3
+```
+PS C:\> $currentJob = Update-AzureRmRecoveryServicesAsrProtectionDirection -AzureToAzure -ProtectionContainerMapping $B2ApcmMapping
```

Comment:
@viverm same comment about adding a trailing backtick to this line to signal continuation on to the next line for the copy-paste scenario